drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -55,6 +55,7 @@ class Hmis::Hud::Client < Hmis::Hud::Base
   has_many :hmis_services, through: :enrollments # All services (HUD and Custom)
   has_many :services, through: :enrollments # HUD Services only
   has_many :custom_services, through: :enrollments # Custom Services only
+  has_many :client_location_histories, class_name: 'ClientLocationHistory::Location'
```

Comment:
This should be added in the driver:
```
drivers/client_location_history/extensions/grda_warehouse/hmis/client_extension.rb
```

Reply:
Thank you @eanders is there any documentation you recommend about Rails drivers? I think there is some railsy magic going on (or rather, trying to go on, and not working correctly on my attempts) that I should understand better.

Reply:
https://github.com/degica/rails_drivers ?

Reply:
That's the one!
https://github.com/degica/rails_drivers

Reply:
Thanks! This ended up working instead: `drivers/client_location_history/extensions/hmis/hud/client_extension.rb`